logging/logs.js

```diff
@@ -0,0 +1,180 @@
+// Copyright 2015-2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START all]
+// [START setup]
+// By default, the client will authenticate using the service account file
+// specified by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use
+// the project specified by the GCLOUD_PROJECT environment variable. See
+// https://googlecloudplatform.github.io/gcloud-node/#/docs/google-cloud/latest/guides/authentication
+var Logging = require('@google-cloud/logging');
+
+// Instantiate a logging client
+var logging = Logging();
+// [END setup]
+
+// [START list_log_entries]
+/**
+ * List log entires in the authenticated project.
+ *
+ * @param {object} [options] Optional. Configuration options.
+ * @param {string} [options.filter] Optional. An advanced logs filter. An empty filter matches all log entries.
+ * @param {number} [options.limit] Optional. Maximum number of logs to return.
+ * @param {string} [options.sort] Optional. Default: "timestamp desc".
+ * @param {function} callback The callback function.
+ */
+function listLogEntries (options, callback) {
+  // Configuration to pass to Logging#getEntries
+  var config = {};
+
+  if (options.filter) {
+    config.filter = options.filter;
+  }
+  if (options.limit) {
+    config.pageSize = options.limit;
```

Comment:
Tried that. gcloud-node throws error if the options contains keys that aren't official options.